libs/react/src/lib/formItem/formItem.tsx

```diff
@@ -140,6 +143,17 @@ export const FormItem = ({
       {children}
       <div className="gds-form-item__footer">
         {validator && <span className="form-info">{validator.message}</span>}
+        {rightAlignedFooterInfo && (
+          <span
+            className="form-info"
+            style={{
+              textAlign: 'right',
+              width: validator ? 'auto' : '100%',
```

Comment:
this makes the text take the entire width when alone but only take the necessary width when displayed together with the validation error